db/migrate/20190829145138_create_data_migrations.rb

```diff
@@ -0,0 +1,5 @@
+class CreateDataMigrations < ActiveRecord::Migration[5.2]
+  def change
+    create_table :data_migrations, primary_key: "version", id: :string unless table_exists? :data_migrations
```

Comment:
I know it's not A Thing in migrations usually, but can we wrap the arguments to `create_table` in brackets, given that we have an `unless` call that the end. Or better still, have a `return if table_exists? :data_migrations` guard clause on the first line